src/dotnet/APIView/ClientSPA/src/app/_components/conversations/conversations.component.html

```diff
@@ -0,0 +1,32 @@
+<h4>Conversations</h4>
+<p-divider />
+<p *ngIf="commentThreads.size === 0">This Review has no Comments</p>
+<p-timeline [value]="getAPIRevisionWithComments()">
+    <ng-template pTemplate="marker" let-apiRevision>
+        <i class="bi bi-clock-history"></i>
+    </ng-template>
+    <ng-template pTemplate="content" let-apiRevision>
+        <div class="pb-3 conversation-group-revision-id" [attr.data-conversation-group-revision-id]="apiRevision.id">
+            <span>
+                <i class="{{ getAPIRevisionTypeClass(apiRevision) }} me-2"></i>
+                <span class="me-2" *ngIf="apiRevision.pullRequestNo"> {{ apiRevision.pullRequestNo }}</span>
+                <span class="emphasis-badge info me-2">version: {{ apiRevision.packageVersion }}</span>
```

Comment:
It could be any. I prefer the non title case
![Conversation](https://github.com/user-attachments/assets/9be64136-a64e-4345-8f62-65c02ea20444)

Reply:
Looking at the image:
- you are right `Created by` is ok to avoid
- `lastUpdated` should be `last updated`
- What is the last value? the name of the review? or the library? adding something there will be good.
- `Source Branch` should be `source branch` -> also, why this one doesn't have a pretty black decorator around it?

Reply:
I have adjusted lastUpdated to last updated. The last value is the review label so I cannot adjust it here.